src/main/java/winstone/AbstractSecuredConnectorFactory.java

```diff
@@ -133,8 +134,9 @@ private static PrivateKey readPEMRSAPrivateKey(Reader reader) throws IOException
                         in = !in;
                         continue;
                     }
+
                     if ( in ) {
-                        baos.write(Base64.getDecoder().decode(line.trim()));
+                        baos.write( Base64.getDecoder().decode(line.getBytes(StandardCharsets.UTF_8) ) );
```

Comment:
Seems unrelated to the basic point of this change. Better suited for a separate PR I think.

Reply:
because I started this long time ago. and this jetty class has been removed in jetty 10 so I had to change it when starting this branch